css/style.css

```
@@ -0,0 +1,143 @@
+@import url("reset.css");
+@import url("variables.css");
+@import url("common.css");
+@import url("layout.css");
+@import url("login.css");
+@import url("signup.css");
+
+
+/* MAIN */
+#cnt{
+  margin-top:7rem;
+  min-height:calc(100vh - 23rem)
+}
+.sec01{
+  display:flex;
+  height:54rem;
+  background-color:#CFE5FF;
+  justify-content:center;
+}
+.sec01>.inner{
+  display:flex;
+  align-items:end;
+  background:url(../img/img_home_top.svg) no-repeat bottom 0 right 0;
+}
+.sec01 .banner_img{
+  width:74.6rem;
+  height:34rem;
+}
+.sec01 .banner_img>img{
+  width:74.6rem;
+  height:34rem;
+}
+
+.sec01 .visual_box{
+  margin-bottom:10rem;
+}
+
+.sec01 .visual_box h2{
+  width:35.7rem;
+  margin-bottom:2.5rem;
+  font-size:4rem;
+  font-weight: bold;
+  line-height:1.4;
+  color:#374151;
```

💊 제안
--color-gray700 변수에 할당한 것과 동일한 값이니 변수로 사용해주세요~

```suggestion
  color: var(--color-gray700);
```